wp-content/themes/goonj-crm/main.js

```
@@ -179,3 +179,164 @@ document.addEventListener("DOMContentLoaded", function() {
 	  }
 	});
 });
+
+document.addEventListener('DOMContentLoaded', function() {
+    var radioButtons = document.querySelectorAll('.crm-contribution-main-form-block .custom_pre_profile-group fieldset .crm-section .content .crm-multiple-checkbox-radio-options .crm-option-label-pair input.crm-form-radio');
+    var recurringSection = document.querySelector('#crm-container #crm-main-content-wrapper form .crm-contribution-main-form-block .crm-public-form-item.crm-section.is_recur-section');
+    var isRecurCheckbox = document.getElementById('is_recur');
+    var installmentsField = document.getElementById('installments');
+    var isRecurCheckbox = document.getElementById('is_recur');
```

Biome (1.9.4)

[error] Shouldn't redeclare 'isRecurCheckbox'. Consider to delete it or rename it.

'isRecurCheckbox' is defined here:


(lint/suspicious/noRedeclare)